conf/test_tsv_fna.config

```diff
@@ -0,0 +1,68 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a fast and simple test. Use as follows:
+ * nextflow run nf-core/eager -profile test, docker (or singularity, or conda)
+ */
+
+params {
+  config_profile_name = 'Test profile'
+  config_profile_description = 'Minimal test dataset to check pipeline function'
+  // Limit resources so that this can run on Travis
+  max_cpus = 2
+  max_memory = 6.GB
+  max_time = 48.h
+  genome = false
+  //Input data
+  input = 'https://raw.githubusercontent.com/nf-core/test-datasets/eager/testdata/Mammoth/mammoth_design_fastq.tsv'
+  // Genome references
+  fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/eager/reference/Mammoth/Mammoth_MT_Krause.fna'
+}
+
+process {
+
+  withLabel:'sc_tiny'{
+      cpus = { check_max( 1, 'cpus' ) }
+      memory = { check_max( 1.GB * task.attempt, 'memory' ) }
+      time = { check_max( 1.h * task.attempt, 'time' ) }
+  }
+
+  withLabel:'sc_small'{
+      cpus = { check_max( 1, 'cpus' ) }
+      memory = { check_max( 4.GB * task.attempt, 'memory' ) }
+      time = { check_max( 1.h * task.attempt, 'time' ) }
+  }
+
+  withLabel:'sc_medium'{
+      cpus = { check_max( 1, 'cpus' ) }
+      memory = { check_max( 8.GB * task.attempt, 'memory' ) }
+      time = { check_max( 1.h * task.attempt, 'time' ) }
+  }
+
+  withLabel:'mc_small'{
+      cpus = { check_max( 2 * task.attempt, 'cpus' ) }
+      memory = { check_max( 4.GB * task.attempt, 'memory' ) }
+      time = { check_max( 1.h * task.attempt, 'time' ) }
+  }
+
+  withLabel:'mc_medium' {
+      cpus = { check_max( 4 * task.attempt, 'cpus' ) }
+      memory = { check_max( 8.GB * task.attempt, 'memory' ) }
+      time = { check_max( 1.h * task.attempt, 'time' ) }
+  }
+
+  withLabel:'mc_large'{
+      cpus = { check_max( 8 * task.attempt, 'cpus' ) }
+      memory = { check_max( 16.GB * task.attempt, 'memory' ) }
+      time = { check_max( 1.h * task.attempt, 'time' ) }
+  }
+
+  withLabel:'mc_huge'{
+      cpus = { check_max( 32 * task.attempt, 'cpus' ) }
+      memory = { check_max( 256.GB * task.attempt, 'memory' ) }
+      time = { check_max( 1.h * task.attempt, 'time' ) }
+  }
+
+}
```

Comment:
You should put all this in a separate config file and call if from all your other `test_*.config` files

Reply:
Done in https://github.com/nf-core/eager/commit/fa5feb4834a92542a97bcabffe80ff5c064ddf31